app/views/layouts/application.haml

```diff
@@ -86,7 +86,7 @@
                 %li= link_to 'Favorites', favorites_path
                 %li= link_to "Post", new_post_path
               = link_to logout_path, method: :delete, id: :"header-logout" do
-                = button_tag "Log out", class: 'button'
+                = button_tag "Log out", class: 'button', :onclick => 'return confirm("Press OK to confirm logout");'
```

Comment:
It looks like [`button_tag` accepts a simple `confirm:` param](https://apidock.com/rails/v6.0.0/ActionView/Helpers/FormTagHelper/button_tag), like `confirm: "Press OK to confirm logout"` instead of this onclick param – do you know if that would work?

Reply:
Might, I'm not sure if the tests will work though - `page.accept_confirm` is specific to the confirm dialogue and I'm not sure if this actually creates a dialogue